feature/bgp/multipath/otg_tests/bgp_multipath_wecmp_lbw_community_test/README.md

```diff
@@ -19,43 +19,117 @@ Validate BGP in multipath UCMP support with link bandwidth community
     *   ATE port-2 and DUT port-2
     *   ATE port-3 and DUT port-3
 *   Enable an Accept-route all import-policy/export-policy for eBGP session
-    under the neighbor AFI/SAFI
-*   Create an IPv4 internal target network attached to ATE port 2 and 3
+    under the neighbor AFI/SAFI - IPv6 unicast and IPv4 unicast.
+*   Create an single IPv4 internal target network attached to ATE port 2 and 3
+*   [TODO] Create an single IPv6 internal target network attached to ATE port 2 and 3
+
 
 ### Tests
 
-*   RT-1.52.1: Verify use of community type
+*   RT-1.52.1: Verify use of unequal community type
+
+    *   Configure ATE port 1, 2 and 3 on different AS, with boths AFI/SAFI
+        * Advertise IPv4 and IPv6 internal target, both, form both ATE port-1 and ATE port-2 in eBGP.
+        * [TODO] For ATE port 2 attach `link-bandwidth:23456:10K` extended-community
+        * [TODO] For ATE port 3 attach `link-bandwidth:23456:5K` extended-community
+    *   Enable multipath, set maximum-paths limit to 2, enable allow multiple
+        AS, and send community type to [STANDARD, EXTENDED, LARGE]
+        *   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/use-multiple-paths/config/enabled
+        *   [TODO] /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/use-multiple-paths/ebgp/config/allow-multiple-as
+        *   [TODO] /network-instances/network-instance/protocols/protocol/bgp/globalp/afi-safis/afi-safi/use-multiple-paths/ebgp/config/maximum-paths
+        *   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/config/send-community-type
+        *   /network-instances/network-instance/protocols/protocol/bgp/global/use-multiple-paths/ebgp/link-bandwidth-ext-community/config/enabled
+    *   Advertise equal cost paths from port2 and port3 of ATE
+    *   Check entries in FIB for advertised prefix, it should have 2 entries.\
+        [TODO] The `weight` leafs of next-hops shall be in 2:1 ratio.
+        *   /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/state/index
+        *   /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/state/weight
+    *   Initiate traffic from ATE port-1 to the DUT and destined to internal
+        target network
+    *   [TODO] Check entire traffic should only be unequally forwarded between DUT
+        port2 and port3 in ratio 2:1
```

Comment:
Can you be more specific?   It's port 2 that should have 66% of traffic and port 3 is 33% of traffic (+/- 5%).  (or your choice of tolerance)